docs/element-definitions.md

```diff
@@ -74,20 +78,33 @@ A less trivial example is the following `asset` Element. The specific semantic i
 
 ## Data Structure Element types
 
+### Values
+Following values can be described in API Elements:
+- null _value_
+- boolean values _true_ and _false_
+- rational numbers, i. e. floating point numbers with finite precision
+- finite character strings
+- finite sets of key value pairs
+- finite lists of values
+
+### Types
```

Comment:
I feel like this whole Type & Predicate introduction should be moved to the top of the document. @kylef What do you think?

Reply:
I agree

Reply:
Also define sub-typing.